lib/include/prjxray/xilinx/xc7series/frames.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PRJXRAY_LIB_XILINX_XC7SERIES_FRAMES_H
+#define PRJXRAY_LIB_XILINX_XC7SERIES_FRAMES_H
+
+#include <string>
+#include <vector>
+
+#include <absl/strings/str_split.h>
+#include <prjxray/xilinx/xc7series/configuration.h>
+#include <prjxray/xilinx/xc7series/frame_address.h>
+#include <prjxray/xilinx/xc7series/part.h>
+
+namespace prjxray {
+namespace xilinx {
+namespace xc7series {
+class Frames {
+       public:
+	typedef std::vector<uint32_t> FrameData;
+	typedef std::map<FrameAddress, FrameData> Frames2Data;
+
+	int readFrames(const std::string& frm_file_str);
+	void addMissingFrames(const absl::optional<Part>& part);
+	Frames2Data& operator()();
```

Comment:
Please just make this just a named method instead of an operator.

Reply:
Ok, this is done.